src/html/partials/problem.html

```diff
@@ -0,0 +1,3 @@
+<section class="problem-section">
+  <h2>Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.</h2>
```

Comment:
Looking at Figma, this looks like an introductory paragraph, but converting it to a `<p>` leaves you without a semantic heading. So this might be a good case for an invisible `<h2>The Problem<h2>` element (for screen readers), followed by a normal paragraph of text. You could just add the h2 for now and add a comment (for later reference) to hide the heading.